crates/sui/src/client_commands.rs

```diff
@@ -2247,3 +2314,86 @@ pub async fn request_tokens_from_faucet(
     }
     Ok(())
 }
+
+fn pretty_print_balance(
+    coins: &BTreeMap<String, (Option<SuiCoinMetadata>, Vec<Coin>)>,
+    builder: &mut TableBuilder,
+    with_coins: &bool,
```

Comment:
Use of `&bool` is unconventional here -- just `bool`.

Reply:
The `with_coins` gets dereferenced later in this function, I guess I can dereference it from the start when this variable is passed.

Reply:
The point here is that it never makes sense to write `&bool` as a parameter -- anything you can do with a `&bool` can be done with a `bool`, and the latter is smaller than the former, so the conventional reasons to pass it by reference do not apply.

You may encounter them in the wild because you are fetching something from some generic interface, instantiated with `bool`, but that shouldn't affect the interfaces that you create.